### ISSUE TITLE
Numbers on the cusp of large spike and small spike returns incorrect prediction

### ISSUE THREAD

**Issue by jdongg** [mikebryant/ac-nh-turnip-prices#356]
**Things to check**
- Have you forced a refresh in your browser (e.g. ctrl+F5 in Chrome/Firefox)?
  - yes
- Have you made sure the first time buyer option is correct? If you're unsure, try it both ways
  - yes
- Have you time-travelled this week? Travelling backwards resets the prices, and therefore you must not put prices in from both before & after the time-travel
  - no

**Describe the bug**
My prices according to the prophet were 100% consistent with a large spike just before the apex of the spike, but it turned out to be a small spike. I believe this may be because my case is on the "cusp" of a large spike and small spike. If I enter my Daisy Price (which was 96), then my prices are considered invalid. If I leave the Daisy price blank, the prophet correctly predicts a small spike AND tells me the predicted range for what my Daisy price was on Sunday is 93-110, which my price of 96 clearly falls into.

**Permalink to your prices**
With Daisy price, prediction of the large spike: https://turnipprophet.io?prices=96.84.79.75.71.66.61.130.144....
With Daisy price, invalid results: https://turnipprophet.io?prices=96.84.79.75.71.66.61.130.144.182...
Without Daisy price: https://turnipprophet.io?prices=.84.79.75.71.66.61.130.144....

**Desktop (please complete the following information):**
 - OS: Mac OS X
 - Browser: Chrome
 - Version: 81.0.4044.129

**Comment by theRTC204** [comment]
Was this the first week someone purchased Turnips on your island?

If it is, then the "First Time Buyer" question should have been marked as Yes. During the first week that turnips are purchased on your island, two things happen.
1. Your pattern for that week is locked to Small Spike.
2. Your prices are regenerated after Daisy leaves on Sunday, meaning the price you paid is no longer relevant; instead, a hidden buy price is used to calculate your sell prices for the rest of that week.

**Comment by jdongg** [comment]
Nope, I've had the game since launch so this is definitely not the first week I've bought turnips! I'm the only resident on the island as well.
